src/vs/workbench/services/positronConsole/browser/positronConsoleService.ts

```diff
@@ -1493,6 +1495,13 @@ class PositronConsoleInstance extends Disposable implements IPositronConsoleInst
 					);
 				}
 
+				// Don't display outputs that are currently handled by the IPyWidgets service.
```

Comment:
I'm not thrilled by this approach.

Maybe we could alternatively extend the `kind` property to all message types supported by Output widgets instead?..

Reply:
In this case of a single instance of something potentially handling the message, I think it's nice an straightforward. I agree that if we start adding other instances where some other part of positron will 'handle' a message it could get a bit spaghetti quickly.

Reply:
I didn't look at this exhaustively, but we generally do want to avoid having to filter the messages like this at every event handler. One way to make this better would be to change the message type/kind upstream (even in the Python extension) to translate it to a message or event specifically handled by the PyWidgets service.